development/docker-compose.postgres.yml

```diff
@@ -0,0 +1,23 @@
+---
+version: "3.8"
+
+services:
+  nautobot:
+    environment:
+      - "NAUTOBOT_DB_ENGINE=django.db.backends.postgresql"
+  db:
+    image: "postgres:13-alpine"
+    env_file:
+      - "development.env"
+      - "creds.env"
+    volumes:
+      # - "./nautobot.sql:/tmp/nautobot.sql"
```

Comment:
remove?

Reply:
I would rather keep it with the template: https://github.com/networktocode-llc/cookiecutter-ntc/blob/nautobot-plugin-v1.0/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/development/docker-compose.postgres.yml#L14

Will probably be removed with the next template mint.

Reply:
It's not gone in `main` either, might want to include it in the cleanup PR if it's not useful at all https://github.com/networktocode-llc/cookiecutter-ntc/blob/main/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/development/docker-compose.postgres.yml